src/main/resources/interface-specification.yml

```diff
@@ -6513,6 +6513,101 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/clients/{clientId}/keys':
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    get:
+      tags:
+        - clients
+      summary: Returns a set of keys by client ID.
+      description: Given a client identifier it returns its corresponding set of keys, if any
+      operationId: getClientKeys
+      parameters:
+        - name: clientId
+          in: path
+          description: ID of the client to look up
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: returns the corresponding array of keys
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReadClientKeys'
```

Comment:
as agreed with Ruggero, we can limit the return payload as
```
type PublicKey = {
  name: string
  createdAt: string
  kid: string
  operator: SelfCareUser
}
```
where `SelfCareUser` has the exact same fields of `OperatorDetails`